team-fingerprints-web/src/test/testHelpers.ts

```diff
@@ -0,0 +1,16 @@
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { JSXElementConstructor, ReactElement } from "react";
+import { BrowserRouter } from "react-router-dom";
+
+export const renderWithRouter = (
+  ui: ReactElement<any, string | JSXElementConstructor<any>>,

```

Comment:
You can just simply do
```suggestion
  ui: JSX.Element
```

Reply:
Right, done